diff --git a/node_modules/@types/plaid-link/index.d.ts b/node_modules/@types/plaid-link/index.d.ts
index bcef922..81aa405 100755
--- a/node_modules/@types/plaid-link/index.d.ts
+++ b/node_modules/@types/plaid-link/index.d.ts
@@ -121,6 +121,7 @@ export namespace Plaid {
         status: ExitStatus;
     }
     interface OnEventMetaData {
+        account_number_mask: string | null;
         error_code: string;
         error_message: string;
         error_type: string;
