diff --git a/node_modules/@types/matter-js/index.d.ts b/node_modules/@types/matter-js/index.d.ts
index 05e8abe..3be3564 100755
--- a/node_modules/@types/matter-js/index.d.ts
+++ b/node_modules/@types/matter-js/index.d.ts
@@ -2870,7 +2870,7 @@ declare namespace Matter {
          * @type {number}
          * @default 1
          */
-        pixelRatio?: number | undefined;
+        pixelRatio?: number | 'auto' | undefined;
     }
 
     interface IRenderLookAtObject {
