diff --git a/node_modules/msw/lib/node/index.js b/node_modules/msw/lib/node/index.js
index 1b341e9..d9879fb 100644
--- a/node_modules/msw/lib/node/index.js
+++ b/node_modules/msw/lib/node/index.js
@@ -62,6 +62,7 @@ var setTimeout = import_timers.setTimeout;
 // src/node/setupServer.ts
 var import_ClientRequest = require("@mswjs/interceptors/lib/interceptors/ClientRequest");
 var import_XMLHttpRequest = require("@mswjs/interceptors/lib/interceptors/XMLHttpRequest");
+var import_Fetch = require("@mswjs/interceptors/lib/interceptors/fetch");
 
 // src/node/createSetupServer.ts
 var import_chalk = require("chalk");
@@ -1353,7 +1354,7 @@ function createSetupServer(...interceptors) {
 }
 
 // src/node/setupServer.ts
-var setupServer = createSetupServer(import_ClientRequest.ClientRequestInterceptor, import_XMLHttpRequest.XMLHttpRequestInterceptor);
+var setupServer = createSetupServer(import_ClientRequest.ClientRequestInterceptor, import_XMLHttpRequest.XMLHttpRequestInterceptor, import_Fetch.FetchInterceptor);
 // Annotate the CommonJS export names for ESM import in node:
 0 && (module.exports = {
   setupServer
